rfcs/0013-get-block-template/0013-get-block-template.md

```diff
@@ -0,0 +1,77 @@
+| work_id               | No       | String           | if provided, this value must be returned with results (see Block Submission) |
+
+#### Transaction Object
+
+The Objects listed in the response's "transactions" key contains these keys:
+
+| Key      | Required | Type             | Description                                                                                                                                                                                                                       |
+| -------- | -------- | ---------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| hash     | Yes      | String           | the hash of the transaction                                                                                                                                                                                                       |
+| required | No       | Boolean          | if provided and true, this transaction must be in the final block                                                                                                                                                                 |
+| cycles   | No       | Number           | total number of cycles, if key is not present, cycles is unknown and clients MUST NOT assume there aren't any                                                                                                                     |
+| depends  | No       | Array of Numbers | other transactions before this one (by 1-based index in "transactions" list) that must be present in the final block if this one is; if key is not present, dependencies are unknown and clients MUST NOT assume there aren't any |
+| data     | Yes      | String           | transaction [CFB][cfb] bytes in  hex-encoded string                                                                                                                                                                               |
+
+### Block Submission
+
+A JSON-RPC method is defined, called `submit_block`. to submit potential blocks (or shares). It accepts two arguments: the first is always a String of the hex-encoded block [CFB][cfb] bytes to submit; the second is String of work_id.
+
+| Key     | Required | Type   | Description                                                           |
+| ------- | -------- | ------ | --------------------------------------------------------------------- |
+| data    | Yes      | String | block [CFB][cfb] bytes in  hex-encoded string                         |
+| work_id | No       | String | if the server provided a workid, it MUST be included with submissions |
+
+
+
+[0]: https://en.bitcoin.it/wiki/Getwork
+[1]: https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_getwork
+[cfb]: https://github.com/nervosnetwork/rfcs/pull/47
```

Comment:
Better use a number instead of `[cfb]` as key to keep consistency with other references.